File: Java/Function.java
public class Function{
    public static void main(String args[])
    {
        Function obj = new Function();
        int x,y,z;
        x=5;y=8;
        z=obj.sub(x,y);
        System.out.println("The sum is "+z);
    }
    int sum(int a, int b)
    {
        return a+b;
    }
    int sub(int a,int b){
        return a-b;
    }
}

File: python/bhawna/data/function.py
class Calculation:

    def __init__(self):
        self.a = 8
        self.b = 9

    def sum(self,b):
        return self.a + self.b+b
    
    def sub(a,b):
        return a-b

    def mul(a,b):
        return a*b



class SQ():
    def square(a):
        return a*a

    def cube(a):
        return a**3


cal = Calculation()
cal.a = 9

print(cal.sum(4))